test/testbool.cpp

```diff
@@ -810,6 +810,20 @@ class TestBool : public TestFixture {
               "}");
         ASSERT_EQUALS("[test.cpp:2]: (style, inconclusive) Boolean expression 'a>0' is used in bitwise operation. Did you mean '&&'?\n", errout.str());
 
+        check("void f(bool a, int b) {\n"
+              "    if(a | b) {}\n"
+              "}");
+        ASSERT_EQUALS(
+            "[test.cpp:2]: (style, inconclusive) Bitwise expression 'a|b' with boolean expression 'a' is equivalent to 'b'?\n",
```

Comment:
I don't understand this. maybe `a` is true and `b` is 0.

Reply:
Ah thats right, then let me remove the warning when using `a|b`.